Bug 965348 - HOST-RESOURCES-MIB::hrFS* not includes btrfs
1006758 - HOST-RESOURCES-MIB::hrFS* not includes btrfs

commit da1fef382591ff45dc92eb3b95a6bfeff9ecfa4f
Author: Jan Safranek <jsafranek@users.sourceforge.net>
Date:   Tue May 21 09:13:41 2013 +0200

    CHANGES: snmpd: Added btrfs support to hrFSTable

diff -up net-snmp-5.7.2/agent/mibgroup/hardware/fsys/fsys_mntent.c.btrfs net-snmp-5.7.2/agent/mibgroup/hardware/fsys/fsys_mntent.c
--- net-snmp-5.7.2/agent/mibgroup/hardware/fsys/fsys_mntent.c.btrfs	2013-09-11 12:12:54.586891414 +0200
+++ net-snmp-5.7.2/agent/mibgroup/hardware/fsys/fsys_mntent.c	2013-09-11 12:12:41.565947166 +0200
@@ -141,6 +141,7 @@ _fsys_type( char *typename )
               !strcmp(typename, MNTTYPE_CVFS) ||
               !strcmp(typename, MNTTYPE_SIMFS) ||
               !strcmp(typename, MNTTYPE_VZFS) ||
+              !strcmp(typename, MNTTYPE_BTRFS) ||
               !strcmp(typename, MNTTYPE_LOFS))
        return NETSNMP_FS_TYPE_OTHER;
 
diff -up net-snmp-5.7.2/agent/mibgroup/hardware/fsys/mnttypes.h.btrfs net-snmp-5.7.2/agent/mibgroup/hardware/fsys/mnttypes.h
--- net-snmp-5.7.2/agent/mibgroup/hardware/fsys/mnttypes.h.btrfs	2013-09-11 12:00:15.283166852 +0200
+++ net-snmp-5.7.2/agent/mibgroup/hardware/fsys/mnttypes.h	2013-09-11 12:00:15.284166847 +0200
@@ -148,6 +148,9 @@
 #ifndef MNTTYPE_VZFS
 #define MNTTYPE_VZFS     "vzfs"
 #endif
+#ifndef MNTTYPE_BTRFS
+#define MNTTYPE_BTRFS    "btrfs"
+#endif
 
 /*
  *  File systems to skip
